### ISSUE TITLE
SiteConfig Request

### ISSUE THREAD

**Issue by dbmtech** [K-vanc/Tempest-EPG-Generator#20]
Hey K-Vanc
I tried a lot to make site config through Temp Wiki Document but didn't succeed. Request you provide at least one real site config making example in Tempset Wiki Document. 
I have another curiosity to know is it possible to make dummy epg through tempset?  

Presently I have two site config making request for Indian Folder.
1. https://www.zee5.com/tvguide
2. https://www.dthhelp.net/epg/epg_channel_list.html

Thanks

**Comment by K-vanc** [comment]
[Ex.zip](https://github.com/K-vanc/Tempest-EPG-Generator/files/10843034/Ex.zip)

**Comment by dbmtech** [comment]
Thanks for explaining in details. I appreciate your effort and don't want to know about encrypted site config making process. I was just curious to know about the simple site config making process. For rest of things like zee5 and d2hhelp site config take your time. I tried webgrab++ but Tempset is really faster also have easy to grabbing epg (specially for new people).
Thanks for your work!

**Comment by dbmtech** [comment]
Sorry for a silly question, is it required separate dummy site config for each dummy channel? Suppose I have 5 channel  which i wan to make dummy epg, so I have to make 5 dummy site config file for each channel.

**Comment by dbmtech** [comment]
ok thanks

**Comment by dbmtech** [comment]
I follow above process, successfully generated multiple channel, but in generated epg file the shows are showing same for all channel. 

I'm attaching file here. Kindly take a look whenever you have free time.

[dummy.zip](https://github.com/K-vanc/Tempest-EPG-Generator/files/10868946/dummy.zip)

Thanks

**Comment by K-vanc** [comment]
hi. Your show pattern was wrong without any channel variable. Also in source file, there should be 7 day blocks for each channel since siteconfig set as all week data there with First day no. I revised both files. I set file destination to my Windows local storage. If you want, modify again or upload to somewhere and change url1.
[dummy_1.zip](https://github.com/K-vanc/Tempest-EPG-Generator/files/10869179/dummy_1.zip)
